sqldeveloper/extension/build.properties

```diff
@@ -6,7 +6,7 @@
 # This is where the builds will deploy the extension
 #      and where we can find our dependencies 
 #sqldev.dir=D:/sqldeveloper-4.2.0.17.089.1709/sqldeveloper
-sqldev.dir=D:/sqldeveloper-18.1.0/sqldeveloper
+sqldev.dir=D:/sqldeveloper-19.2.1/sqldeveloper
 
 # Information about the company or person creating the extension
 extension.owner=Oracle Examples

```

Comment:
Seems like you have MS-DOS carriage returns in that file but in no others.
Perhaps you should convert it into UNIX as well just to keep it consistent.